.changeset/twelve-dragons-reply.md

```diff
@@ -0,0 +1,5 @@
+---
+"live-mobile": major
```

Comment:
⚠️ this should not be a major bump, could you change it to minor?

Reply:
Thanks! Done!